.gitattributes

```diff
@@ -0,0 +1,9 @@
+* text eol=lf
```

Comment:
Force using the proper line endings on checkout.  This solves tons of errors when checking out the branch on a windows machine, because the bash scripts get corrupted with CRLF line endings.

Note that I've also added a few types to be treated as binary so they are not corrupted either.